app/controllers/application_controller.rb

```diff
@@ -189,6 +189,10 @@ def set_lti_launch_values
     @is_lti_launch = true
     @canvas_url = current_application_instance.site.url
     @app_name = current_application_instance.application.client_application_name
+    context_id = params[:context_id]
+    if current_user&.can_author?(context_id, current_application_instance)
+      @can_author = true
```

Comment:
This seems like it may be AA specific that accidentally got merged in upstream? I can see the method exists on the user as well, which is kinda weird.

Reply:
Here it means the lti roles are teacher or admin, which I'm using to determine when the user can resolve a launch issue.  There might be a better way to do this.